apps/authentication-portal/src/main/webapp/login.jsp

```diff
@@ -408,17 +408,42 @@
                                             </form>
 
                                             <script>
-                                                window.onload = function callGoogleOneTap() {
-                                                    google.accounts.id.initialize({
-                                                        client_id: "<%=Encode.forJavaScriptAttribute(GOOGLE_CLIENT_ID)%>",
-                                                        prompt_parent_id: "google_parent",
-                                                        cancel_on_tap_outside: false,
-                                                        nonce: "<%=Encode.forJavaScriptAttribute(request.getParameter("sessionDataKey"))%>",
-                                                        callback: handleCredentialResponse
-                                                    });
-                                                    google.accounts.id.prompt((notification) => {
-                                                        onMoment(notification);
-                                                    });
+                                                if (navigator) {
+                                                    let userAgent = navigator.userAgent;
```

Comment:
Let's use ES5 syntax since these snippets run on the browser without any compilation.

You can use TS playground to convert ES6 code to ES5.

https://www.typescriptlang.org/play?target=1&ts=4.9.3&ssl=36&ssc=2&pln=2&pc=5#code/JYMwBAFAdghgbsA5jALgewE4EowG8BQYRYANgKYpgCuAzmRgIKJlSUC8YsCy6GAdLXpMWKANyFi5SgCMMaAO50MAORgBbMuOKkKYDGRooMwAMYoyAEwBCcxfRoAxTAHEA8gBUwHAEQAeAKRs+obGZpY2CkqOLh7+AHzeWsSgkIKMzKx8aqgmABYQAPR5choAPsVoasBUauXGaDQFwFg4BNraspH0qhpeYN4VGokSRAC+YGQkdGApEGnCmdkoeYUgwPogaAAepSBbwA1NLXgj7Z12Kupkfd5rG9vD2uOT07PzGShZOfkFNDAgMGMR1apw6tiUPWuPj+AOMj2Izym1zeSgWnyWKwKaAADhgADoFYEndrEc4Qq43HH0GDwsYTJEzcBzVEfL7LH6WRBEtoksBk7oUnycsi0sCI6Y8kn8y69HzKNB88H0MAWChkMwHKCi0anWbBIymczWJUYaIYNyeACEbA4UCoJBIoOIADJnXoDAawsauqanOaPGBrRwAOTBp1EV3ukKG8Ims0WvjoAAyCnoAGEYHQIFg+MAoCYSFRVTQINLIcdJe0LGgTDURHxmCgAKLkDSsKwATwAkhYIN5EGg0IhyABlJBQLtanOGDvkPgWYA0bEkGAdm7SEg1gDW2vpEvDYHkeer8j4aCgm5gFj6ICo+ZQmrAJhgDucg+HZFcUDI7hg2OzxK8toA5DnOMAmCYaB3igNC5hYuZQMAD4vsAABeZAQJWQHaAWwAiAA+sAFgAFz9AEbBNvmaCqnwmwYAAUvAMAjiYxjYigDAoAa0hUOYEBuK4zhJk2+FpkmXZNso7j4V2AAiWDxN4AA0B5AbilTsfh2KAgRRGkf277kFpOmsMpqm8s++aTPh574Sgf42bxNBEWQpEAkiKnYbyUDniYrlkYElGQTRdGMXAzGscA7GcdxvEYfoACOVAeg2FAAAqAlc5gYH2dA0M556yagMAANJkB23gtIpnlee0z4OtI4FbqRuQwFAFjkGm+iqqwwAvgASgY2LnnQ5ljFgSReSBH58OBkHQbBRF8OpajsRA0BoA+azPg+544GwcSAbVxDngAspUIgbVtpioJqE1jWK91ATqTz4KMQA
```suggestion
                                                    var userAgent = navigator.userAgent;
```

Reply:
This is fixed 0a74fc9395627a6aff8eae85e493f5544941ead4